src/pages/candidate-details/candidate-details.html

```diff
@@ -139,16 +137,37 @@ <h4>Contact details</h4>
           </ion-col>
         </ion-row>
         <ion-row class="mes-data-row">
+          <ion-col col-40>
+            <label>Address</label>
+          </ion-col>
+          <display-address [address]="(pageState.details$ | async).address"></display-address>
+        </ion-row>
+      </ion-grid>
+    </ion-card-content>
+  </ion-card>
+  <ion-card *ngIf="(pageState.business$ | async)">
+    <ion-card-header>
+      <h4>Business contact details</h4>
+    </ion-card-header>
+    <ion-card-content>
+      <ion-grid>
+        <ion-row *ngIf="(pageState.business$ | async).telephone" class="mes-data-row-with-separator">
+          <ion-col col-40>
+            <label>Phone number</label>
+          </ion-col>
+          <ion-col col-56>
+            <span class="mes-data">{{ (pageState.business$ | async).telephone }}</span>
+          </ion-col>
+        </ion-row>
+        <ion-row *ngIf="(pageState.business$ | async).businessAddress" class="mes-data-row-with-separator">
           <ion-col col-40>
             <label>Address</label>
           </ion-col>
           <ion-col col-56>
-            <div class="mes-data">{{ (pageState.details$ | async).address.street }}</div>
-            <div class="mes-data">{{ (pageState.details$ | async).address.city }}</div>
-            <div class="mes-data">{{ (pageState.details$ | async).address.postcode }}</div>
+            <display-address [address]="(pageState.business$ | async).businessAddress"></display-address>
```

Comment:
We will need tests for the new component. Especially as the redundant test for 'Address' has been removed above.

Reply:
Yeah just adding tests now